landing-page/content/common/release-notes.md

```diff
@@ -70,8 +70,21 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 
 ## 1.1.0 release
 
+Has been released on 28 November 2022. The 1.1.0 release deprecates various pre-1.0.0 methods:
+
+* Puffin statistics have been [added to the Table API](https://github.com/apache/iceberg/pull/4945)
+* Support for [Table scan reporting](https://github.com/apache/iceberg/pull/5268), which enables collection of statistics of the table scans.
+
```

Comment:
On top of my head these are worth mentioning:
1. Add dataSequenceNumber to ManifestEntry #5913
2. Support register table for all the catalogs (previously it was only for hive) #5037
3. Flink 1.16 support, flink 1.13 removal 
4. Spark-3.0 removal

Reply:
Great suggestions, added 👍🏻